llvm/lib/IR/Verifier.cpp

```diff
@@ -2314,7 +2314,7 @@ void Verifier::verifyFunctionAttrs(FunctionType *FT, AttributeList Attrs,
 
   if (auto A = Attrs.getFnAttr("sign-return-address"); A.isValid()) {
     StringRef S = A.getValueAsString();
-    if (S != "none" && S != "all" && S != "non-leaf")
```

Comment:
This isn't right, to support\import the old format the old values and nothing need to accepted here.